rpa-bot/1.6/modules/ROOT/pages/_partials/hard-soft-requirements.adoc

```diff
@@ -10,6 +12,26 @@
 | 30 GB of disk space | 30 GB (SSD) of disk space
 |===
 
+* To run more than one process in parallel, when using Secure Sessions, you need to add 2 GB of RAM plus the RAM needed to run the process for each parallel run to these recommended values.
++
+For example, to run two different processes in parallel:
++
+Process 1 requires one GB of RAM and runs 10 times in parallel.
++
+Process 2 requires three GB of RAM and runs 5 times in parallel.
```

Comment:
```suggestion
Process 2 requires three GB of RAM and runs five times in parallel.
```

Reply:
I gave you bad advice by calling these "Process 1". Maybe Process A and Process B?